dashboard-api/internal/platform/server/handler/studies/list.go

```diff
@@ -0,0 +1,113 @@
+package studies
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	studiesmanager "github.com/vlab-research/vlab/dashboard-api/internal"
+)
+
+type studyReponse struct {
```

Comment:
You have a Study and a studyResponse that are the exact same. If you want an abstraction, why not just use a type alias and then if they ever do differ, someone can change it.

Reply:
For now I'll just return the entities, no extra type or type alias. Just like in the other PR -> https://github.com/vlab-research/vlab/pull/55#discussion_r743382106